sdk/cosmos/azure-cosmos/src/main/java/com/azure/cosmos/implementation/directconnectivity/Uri.java

```diff
@@ -195,10 +195,10 @@ public String toString() {
      * </p>
      */
     public enum HealthStatus {
-        Connected(0),
-        Unknown(1),
-        UnhealthyPending(2),
-        Unhealthy(3);
+        Connected(100),
```

Comment:
why are we changing these?
To be able to add new HealthStatus in future if we need to?

Reply:
This is changed based on the previously discussion we had --> the priority of each health status is important, using a larger value to give flexibility in the future if we want to add new status in between